src/lib/client.h

```diff
@@ -58,7 +58,11 @@ class Client : public QObject
 	QFuture<ds3_get_bucket_response*> GetBucket(const QString& bucketName,
 						    const QString& prefix,
 						    const QString& marker,
-						    bool silent = false);
+						    bool silent = false,
+						    const QString& delimiter = "/");
+	QFuture<ds3_get_objects_response*> GetObjects(const QString& bucketName,
```

Comment:
The `ds3_get_objects_response` struct hasn't been merged into the C SDK yet, so we will want to do that first before this gets merged.